packages/libs/src/sdk/api/chats/ChatsApi.ts

```diff
@@ -282,6 +289,23 @@ export class ChatsApi
     }
   }
 
+  /**
+   * Gets a list of chat blasts for which chats haven't been created yet
+   * @returns the blast messages list response
+   */
+  public async getBlasts(): Promise<TypedCommsResponse<ChatBlast[]>> {
```

Comment:
nit: maybe this should be private since it's not meant to be called externally? also maybe worth naming/documenting that this is _received_ blasts, not sent blasts